app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const fs = require("fs/promises");
+
+const {
+  topicsRoute
+} = require("./db/controllers/data.controller.js")
+
+const app = express();
+
+app.get("/api/topics", topicsRoute);
+
+app.listen(9090, (err) => {
+  if (err) console.log("Error running on server")
+  else console.log("Server running on port 9090")
+})
```

Comment:
Should your app.listen live in your app? Think about extracting this out into it's own listen.js file 🚀